### PR TITLE
Fix upgrade for IBM Cloud ROKS cluster

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #4726

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/ocs_upgrade.py [hunk 1]
```diff
@@ -505,6 +505,20 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
         channel = upgrade_ocs.set_upgrade_channel()
         upgrade_ocs.set_upgrade_images()
         upgrade_ocs.update_subscription(channel)
+        if (config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM) and not (
+            upgrade_in_current_source
+        ):
+            log.info("Sleep 30 mins before linking secret/SAs")
+            time.sleep(30)
```
Comment (Avilir) [inline ocs_ci/ocs/ocs_upgrade.py:512]: did you meant 30 sec. or the sleep need to be 1800 ?

Comment (petr-balogh) [reply ocs_ci/ocs/ocs_upgrade.py:512]: 30 secs :) thanks for catching that, will update

**Comment by petr-balogh** [comment]
Verified here: https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/5431/

**Comment by petr-balogh** [comment]
Verified here:
https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/5431/
and here:
https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/5429/

**Review by dahorak — APPROVED** [review]

**Review by Gangadhar-ibm-lab — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**